controllers/articles.controllers.js

```diff
@@ -1,5 +1,14 @@
-const {fetchArticle} = require('../models/articles.models')
+const {fetchArticle, fetchAllArticles} = require('../models/articles.models')
 
+exports.getAllArticles = (req, res, next) => {
+    fetchAllArticles()
+    .then((response) => {
+        res.status(200).send(response)
+    })
+    .catch((err) => {
```

Comment:
Here is where there is a slight mistake on the response, when sending the response the data should be stored in an object, this can be done like so:
`.send({ articles: response });`